TTD/docs/TTDLiveRecorder.h/ILiveRecorder_ResumeRecording.md

```diff
@@ -0,0 +1,40 @@
+# TTD::ILiveRecorder::ResumeRecording method
+
+Method of the [`TTD::ILiveRecorder`](interface-ILiveRecorder.md) interface.
+
+Resumes recording the current island after the recording was temporarily paused.
+
+If the recording is not paused (either because the current thread is recording or because it never started recording
+or because recording was stopped) this method does nothing.
+
+```C++
+bool TTD::ILiveRecorder::ResumeRecording() noexcept;
+```
+
+## Return value
+
+Returns `true` if the current thread is being recorded when the method returns.
+
+## Correct use
+
+This method only affects the recording of the calling thread and may be freely called from multiple threads simultaneously.
+
+This method must only be called after a call to [`TTD::ILiveRecorder::TryPauseRecording`](ILiveRecorder_TryPauseRecording.md)
+that returned `true`, signifying a successful pause of the recording.
+
+It is highly recommended to use the [`TTD::ILiveRecorder::ScopedPauseRecording`](ILiveRecorder_ScopedPauseRecording.md) class
+instead of calling this method directly.
+
+## Example of use
+
+```C++
+bool const wasRecording = pRecorder->TryPauseRecording();
+
+// Recording is definitely paused now.
+// We can do stuff that we don't want in the recording.
```

Comment:
commentary: it's a little confusing to "Try" PauseRecording() and then declare we definitely paused recording.

suggestion: change the comment to something like "Recording is definitely not active now", or "Recording (if any) is paused"

Reply:
"(if any)" is a good clarification here. "not active" is confusing, considering that we call this "paused" everywhere.

Do you think the API should be called `PauseRecording`? That'd be ok.

Reply:
TryX() with bool return is standard issue API design, I'm ok with that. The slight caveat is TryX() typically is for something that is ok to fail, there's really nothing that fails here.

Don't think we need to change the API name.